src/runtime/auto-prompt-manager.js

```diff
@@ -44,6 +45,23 @@ const dismissEvents = [
   AnalyticsEvent.ACTION_CONTRIBUTION_OFFERS_CLOSED,
   AnalyticsEvent.ACTION_SUBSCRIPTION_OFFERS_CLOSED,
 ];
+/** @const {!Array<!AnalyticsEvent>} */
+const completedActions = [AnalyticsEvent.ACTION_SURVEY_SUBMIT_CLICK];
```

Comment:
Would `ACTION_SURVEY_DATA_TRANSFER` be more accurate to determine when it's completed?